snap/README.md

```diff
@@ -0,0 +1,149 @@
+# EdgeX Grove Device Service Snap
+[![snap store badge](https://raw.githubusercontent.com/snapcore/snap-store-badges/master/EN/%5BEN%5D-snap-store-black-uneditable.png)](https://snapcraft.io/edgex-device-grove)
+
+This folder contains snap packaging for the EdgeX Grove Device Service Snap
+
+The snap currently supports `arm64` platform.
+
+## Installation
+
+### Installing snapd
+The snap can be installed on any system that supports snaps. 
+You can see how to install snaps on your system [here](https://snapcraft.io/docs/installing-snapd/6735).
+
+However for full security confinement, the snap should be installed on an
+Ubuntu 18.04 LTS or later (Desktop or Server), or a system running Ubuntu Core 18 or later.
+
+### Installing EdgeX Device Grove as a snap
+The snap is published in the snap store as [edgex-device-grove](https://snapcraft.io/edgex-device-grove).
+You can see the current revisions available for your machine's architecture by running the command:
+```bash
+snap info edgex-device-grove
+```
+
+The latest beta version of the snap can be installed using:
+```bash
+
+sudo snap install edgex-device-grove --beta
+```
+
+The latest development version of the snap can be installed using:
+```bash
+sudo snap install edgex-device-grove --edge
+```
```

Comment:
I suggest removing the risk-level to simplify future maintenance. The snap cli returns appropriate instructions:
```
$ sudo snap install edgex-device-grove
error: snap "edgex-device-grove" is not available on stable but is available to install on the
       following channels:

       beta       snap install --beta edgex-device-grove
       edge       snap install --edge edgex-device-grove

       Please be mindful pre-release channels may include features not completely tested or
       implemented. Get more information with 'snap info edgex-device-grove'.
```

Reply:
Thanks. Updated!